docs/src/install/plugins.rst

```diff
@@ -0,0 +1,35 @@
+***********************
+Installation of plugins
+***********************
+
+Oríon is built to be easily extensible. Algorithms and database backends can be implemented in
+external repositories and installed as plugins. The installation process is very simple, you only
+need to install them as you would for Oríon's core (see :doc:`/install/core`). They will be
+automatically detected and available for Oríon at run-time.
+
+Note that active development is currently focused on the core. Therefore only one algorithm plugin
+is available for now.
```

Comment:
We also have gradient descent inside the tests, should we move it to its own repo?